hypha/apply/projects/templates/application_projects/project_detail.html

```diff
@@ -214,6 +214,13 @@ <h5>{% trans "Contracts" %}</h5>
                             - {% trans "Approved by" %} {{ contract.approver }}
                     </p>
                     {% endfor %}
+
+                    {% block correction_in_contract %}
+                        <a class="link link--bold" href="#communications" target="_blank">
+                            <svg class="icon icon--info-circle"><use xlink:href="#info-circle-fill"></use></svg>
+                                <u>Corrections or amendments to your contract?</u>
```

Comment:
I think we should use "span" and not "u" tag here.

Reply:
@frjo I used "u" just to underline the link as per the design. If underlining is not necessary then yes "span" would be a better option.